powertools-e2e-tests/handlers/logging/pom.xml

```diff
@@ -15,7 +15,22 @@
     <dependencies>
         <dependency>
             <groupId>software.amazon.lambda</groupId>
-            <artifactId>powertools-logging</artifactId>
+            <artifactId>powertools-logging-log4j</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
```

Comment:
not sure all examples/e2e tests are aligned on this, I need to check.

@scottgerring, Question by the way: should we provide them as dependency of the `powertools-logging-log4j` module or ask the customers to add them (so they can add their own versions?)

Reply:
@jeromevdl I was wondering this myself. I think you will have a better feeling for this than I do. We had another issue recently that was the same pattern, no? - rather than concretely providing a version, take it from the embedding project. It was a user raised issue I believe - struggling to find the ticket.